scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -1,5 +1,5 @@
 /* exported updatedChromSelOptions */
-function updatedChromSelOptions() {
+function updatedChromSelOptions() { // eslint-disable-line no-unused-vars
```

Comment:
👍  Nice! Several of the other "unused" new and old functions work with commenting them as "exported", and the warnings line numbers were off so I kind of figured this would resolve once the linter started tracking again. 😊

Reply:
I just found it!

You can also skip the linter check on blocks of code like this:

/* eslint-disable */ 

All the code you want

/* eslint-enable */

Reply:
Yeah, good job! I vaguely recognise it from last time. It is kind of similar to the  `#noqa`-tags! It is also supposed to be good with that `/* exported NAME */`. But not so this time, or at least not for all of the functions.. 🤷